e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -1242,31 +1270,71 @@ describe("scenarios > embedding > full app", () => {
         });
       });
 
-      it("should be able to join a card when the data source is a table", () => {
-        const cardDetails = {
-          ...ordersCardDetails,
+      it("should join a table when the data source is a model", () => {
```

Comment:
ok, so the diff is super funny. Anyways, the test that were here was moved up above since I put it in the wrong place. It was a table joining a model, so the test should be under `"table"` not `"model"`


And 2 new tests were added.
1. model joins table
2. model joins model